src/components/Navbar/Account.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+
+interface NavbarAccount {
+  isUserExist?: boolean
+}
+
+const NavbarAccount = ({ isUserExist }: NavbarAccount) => {
+  const router = useRouter()
+
+  const handleSignInClick = () => {
+    // TODO: Open SignIn modal
+  }
+
+  const handleSignUpClick = () => {
+    router.push('/sign-up')
+  }
+
+  const handleAvatarClick = () => {
+    // TODO: Expand dropdown or sth
+  }
+
+  return (
+    <div className="navbar__account">
+      {isUserExist ? (
+        <button className="navbar__avatar-button" onClick={handleAvatarClick}>
```

Comment:
I don't think avatar is a button. It is just an image displaying the user profile. How a normal `div` for this?

Reply:
But if it's going to be dropdown then I understand the usage of `button`. But then is it really `button`?

Reply:
Yep, it's going to be a dropdown button. Since we are going to import this component from `bootstrap` or `material-ui` Can we keep only a button for now?